src/pwacompat.js

```diff
@@ -124,7 +150,25 @@ function unused() {
     return (part) => part || '';
   }
 
+  /**
+   * Adds an element in the <head> if it's not present already
+   * nb: we check, but this won't override any _earlier_ (in DOM order)
+   * @param {string} localName tag name
+   * @param {*} attr key-value collection of attributes
```

Comment:
This would actually be type `{!Object<string>}`. It might help the compiler 🤷

Reply:
done ;)